config/networking/network-policies.yaml

```diff
@@ -721,3 +721,20 @@ spec:
       namespaceSelector:
         matchLabels:
           cf-for-k8s.cloudfoundry.org/istio-system-ns: ""
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: eirini-app-migration
+  namespace: cf-system
+spec:
+  podSelector:
+    matchLabels:
+      name: eirini-app-migration
+  egress:
+  - {}
+  ingress:
```

Comment:
Why is ingress being allowed?

Reply:
I think this is a mistake, I'm pretty sure we don't need it. I'll fix it ASAP.

Reply:
Done!